### DISCUSSION TITLE
Customizing items in grid and editing filters

### DISCUSSION THREAD

**Discussion by KyleKun** [fluttercandies/flutter_wechat_assets_picker#484] [Q&A] [answered]
Hi, thanks for your work!

I have two questions:

1) Can I add a text below each item in the picker grid with the file date? If so, how do I retrieve the date?

2) Can I edit filters inside the picker? Example, I open it with :

```
final FilterOptionGroup filterOptionGroup = FilterOptionGroup(
        containsPathModified: true,
        updateTimeCond: DateTimeCond(
          min: _selectedDate,
          max: DateTime.now(),
          ignore: false,
        ),
        orders: [
          OrderOption(
            type: OrderOptionType.updateDate,
            asc: true,
          ),
        ],
      );
```

and would like to have a button to make this filter stop and show all videos. Like setting `ignore` to `true`. Is this possible?

Thank you!

**Comment by AlexV525** [comment, marked as answer]
1. https://pub.dev/documentation/photo_manager/latest/photo_manager/AssetEntity/createDateTime.html
2. https://github.com/fluttercandies/flutter_wechat_assets_picker/tree/main/example/lib/customs

**Reply by KyleKun** [reply]
Thanks a lot! I ended up making a custom fork to implement these and it works perfectly for my [video diary app](https://github.com/KyleKun/one_second_diary) usecase.
